cmd/podsync/updater.go

```diff
@@ -103,13 +110,6 @@ func (u *Updater) Update(ctx context.Context, feedConfig *config.Feed) error {
 			logger.Debug("skipping download of episode")
 		}
 
-		// Record file size
-		if size, err := u.fileSize(episodePath); err != nil {
-			return errors.Wrap(err, "failed to get episode file size")
```

Comment:
We still need this to get file size for existing episodes (the ones, that we didn't download during this update). 

I suggest something like this instead:
```suggestion
// Don't return on error, use estimated file size provided by builders
logger.WithErr(err).Error("failed to get episode file size")
```

Reply:
You are absolutely right (and you know your project way better than I). I added this last night to have a podcast working before hopping on the subway for my commute. I'll update the PR.